valueformatting.go

```diff
@@ -161,8 +163,56 @@ func (f *valueFormatting) binaryFormatter(
 	}
 }
 
+// Returns a valueFormatter function that pretty-prints JSON values.
+func (f *valueFormatting) jsonFormatter() (valueFormatter, error) {
+	return func(in []byte) (string, error) {
+
+		var outputJSONInterface interface{}
+		err := json.Unmarshal(in, &outputJSONInterface)
+		if err != nil {
+			return "", err
+		}
+
+		// Recursive inner function that returns strings from
+		// JSON values and nested JSON data structures
+		var fmat func(v interface{}) string
+		fmat = func(v interface{}) string {
+			switch t := v.(type) {
+			case string:
+				return fmt.Sprintf("%6s", fmt.Sprintf("\"%s\"", t))
```

Comment:
I think this can just be:

``` golang
return fmt.Sprintf("%6q", t)
```

`q` will give you the quotes, and the spacing won't include them. https://go.dev/play/p/EJpHMkL8Wbx

Reply:
Done